### ISSUE TITLE
iOS 16.6+ support

### ISSUE THREAD

**Issue by KpwnZ** [KpwnZ/Def1nit3lyN0tAJa1lbr3akTool#16]
Our first priority is supporting iOS 16.6+ now.

**Comment by nyathea** [comment]
Progress on 16.6 support?

**Comment by KpwnZ** [comment]
> Progress on 16.6 support?
KPF now fully supports iOS 16.6. And you only need to import landa exploitation to make it work on iOS 16.6.

**Comment by Zhou-Shilin** [comment]
Progress on arm64e support?
(16.2 beta2, trollstore&bootstrap installed, m1)

**Comment by KpwnZ** [comment]
> Progress on arm64e support? (16.2 beta2, trollstore&bootstrap installed, m1)

We have krw now.

**Comment by KpwnZ** [comment]
arm64e: we have PACless kernel memory allocating and physical memory access now. Jailbreakd can run on arm64e devices with krw primitive now.

**Comment by 1ahmadbassam** [comment]
Can you rephrase that in more JB-user friendly terms? Thanks.

**Comment by Florian-Martinez** [comment]
Is there any eta for this?

**Comment by KpwnZ** [comment]
> Is there any eta for this?

1 to 3 weeks I guess. But I want to test an experimental pac bypass.

**Comment by imomao** [comment]
will this ever work on 16.7+?

**Comment by nyathea** [comment]
> will this ever work on 16.7+? 

No

**Comment by NoW4U2Kid** [comment]
> > Is there any eta for this?
> 
> 1 to 3 weeks I guess. But I want to test an experimental pac bypass.

Will this work on iOS 16.6.1 Arm64e, or is it only up to 16.5.1?

**Comment by phoen1x74** [comment]
will the commits for this all be public?

**Comment by ITProggeruz** [comment]
> > > Is there any eta for this?
> > 
> > 
> > 1 to 3 weeks I guess. But I want to test an experimental pac bypass.
> 
> Will this work on iOS 16.6.1 Arm64e, or is it only up to 16.5.1?

If they doesnt i would ever get emotional damage
Alsk its me in discord cypwn

**Comment by 1ahmadbassam** [comment]
Any updated progress?

**Comment by FStechek** [comment]
Elo?

**Comment by FStechek** [comment]
Eta for ifon (ipon) 11 ios 16.6.1?

**Comment by ITProggeruz** [comment]
I dunno

Вт, 27 февр. 2024 г. в 20:52, FStechek ***@***.***>:

**Comment by ip146** [comment]
Do you support ios14?

**Comment by jthoward64** [comment]
> Do you support ios14?

No

**Comment by ITProggeruz** [comment]
any eta or the creator metafarted
